### DISCUSSION TITLE
Why am I ranked higher than Mephisto here?

### DISCUSSION THREAD

**Discussion by zhillyz** [Heroes-Profile/heroesprofile#961] [Q&A] [answered]
![image](https://github.com/Heroes-Profile/heroesprofile/assets/46723600/90590fda-8598-49e4-a3bf-1f7ce18a82b2)

**Comment by Zemill** [comment]
Its hard to answer that given the information provided.   What replay is this

**Reply by zhillyz** [reply]
Sorry I thought my name matching between GitHub and HoTS was enough haha. I am the LiMing it is a SL game, and looking at every individual stat I seem to be worse than him but the score shows me as 1 point better. Are there hidden metrics people are scored on or is there some glitch?

**Comment by Zemill** [comment]
There are some metrics that go into the score that are not shown in that section.  Same as Heroes of the Storm's MVP/Award system

See Kills, Assists, Takedowns, Deaths, Time Spent Dead, Experience, Siege Damage, Hero Damage, Healing, Self Healing, Mercs, Watch Tower Captures, and Damage Taken.

Next time please provide the replayID.  It may not be so easy to find a game based on the information you provided.  For example, I imagine it would take you some time to find this game

![image](https://github.com/Heroes-Profile/heroesprofile/assets/33404722/d4afabce-0d50-49fb-a1f7-b33e80465d33)

**Reply by zhillyz** [reply]
Haha I mean, you are correct I wouldn't be able to identify that game! I naively thought that things would be calculated from the inimage numbers only and you would be able to tell me something was wrong. Sidenote: cool no damage taken metric on Aba!

Please see the game ID below which should hopefully enable you giving me the reason i ranked higher!
[52855633](https://www.heroesprofile.com/Match/Single/52855633).

Thank you for your time :)

**Comment by Zemill** [comment, marked as answer]
You spent less time dead, had more merc camp captures. 

Lets use a simplified example.

Azog
Kills - 1st - 10 pts
Hero Damage - 1st - 10 pts
Time Spent Dead - Last - 0 pts
Merc Camp Captures - 5th - 6 pts

Total Score = 26

Zhillyz
Kills - 2nd - 8 pts
Hero Damage - 2nd - 8 pts
Time Spend Dead - 5th - 6 pts
Merc Camp Captures - 1st - 10pts

Total Score = 32


Thats a super oversimplified example, but that should show it a bit

**Reply by zhillyz** [reply]
Thank you so much buddy, very interesting. Would love a deep dive button on the website that showed you these extra merits. I'm more of a backend python developer than front end JavaScript/React but would volunteer some time if I can find it with work and family commitments 😅

**Reply by Zemill** [reply]
If you scroll down on the page you can see these extra metrics in the advanced stats section

**Reply by zhillyz** [reply]
You're on a roll of being super helpful and I'm on a roll of being super oblivious. Thanks again 🙏

**Reply by Zemill** [reply]
Also,
https://api.heroesprofile.com/Api

https://api.heroesprofile.com/docs/1.0/Replay/Data

If you really want to dig deeper into the data we offer an API
